### ISSUE TITLE
.nsgrc causes error on windows

### ISSUE THREAD

**Issue by HansHammel** [samueleaton/npm-scripts-gui#6]
i get this, wenn a .nsgrc file is presen

```
C:\project>nsg

exception

ReferenceError: configName is not defined
```

**Comment by samueleaton** [comment]
It looks like your .nsgrc was not proper JSON. Either its not formatted properly or its an empty file. You would need to at least have an empty object `{}`.

I pushed a commit with better error output. It will be available the next time I publish the module to NPM.

**Comment by samueleaton** [comment]
Fixed 6da8978ec4c96f8c49fea82b460f70e583824097
